code/modules/mob/transform_procs.dm

```diff
@@ -49,7 +49,7 @@
 	QDEL_NULL_LIST(worn_underwear)
 	return ..(move)
 
-/mob/living/carbon/AIize()
+/mob/living/AIize()
```

Comment:
should probably do an early return for `/mob/living/silicon/ai/AIize()`? or maybe in that case it's actually desirable to create a new AI mob. if so, ignore